python/plugins/processing/core/ProcessingConfig.py

```diff
@@ -83,6 +84,10 @@ def initialize():
             ProcessingConfig.tr('Use only selected features'), True))
         ProcessingConfig.addSetting(Setting(
             ProcessingConfig.tr('General'),
+            ProcessingConfig.IGNORE_NULL_GEOMS,
+            ProcessingConfig.tr('Ignore features with null geometries'), True))
```

Comment:
Great, thanks.

That wasn't related to performance in a speed sense, more in a functionality sense. If my algorithm now properly handles null values, it would no longer do after this change until I specify `ignoreNullGeoms=False`.